src/components/TextArea.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FC, HTMLAttributes, useState } from "react";
+
+import { StyleSheet, css } from "aphrodite";
+import { colors } from "styles/palette";
+
+interface Props {
+  onChange: (_value: string) => void;
+  value: string;
+  error?: string;
+  focused?: boolean;
+  onBlur?: () => void;
+  onFocus?: () => void;
+  label?: string;
+  placeholder?: string;
+  style?: HTMLAttributes<HTMLInputElement>;
+}
+
+const TextArea: FC<Props> = ({
+  error,
+  focused = false,
+  label,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder,
+  style,
+  value,
+}) => {
+  const [isFocused, setIsFocused] = useState<boolean>(focused);
+
+  const handleFocus = () => {
+    setIsFocused(true);
+    onFocus?.();
+  };
+  const handleBlur = () => {
+    setIsFocused(false);
+    onBlur?.();
+  };
+
+  return (
+    <>
+      <label htmlFor={label}>
+        {label}
+        <textarea
+          id={label}
+          onFocus={handleFocus}
+          onChange={(event) => onChange(event.target.value)}
+          onBlur={handleBlur}
+          className={css(
+            styles.textarea,
+            isFocused && styles.focused,
+            style,
+            !!error && styles.error
+          )}
+          placeholder={placeholder}
+          value={value}
+        />
+      </label>
+      {!!error && <p className={css(styles.errorText)}>{error}</p>}
```

Comment:
This paragraph is the same as in the `StyledInput` component so please create an addtional component `<StyledText />` and replace those paragraphs.

Reply:
Also in the future this `StyledText` component should contain ur new default fonts